src/main/java/com/facebook/openwifirrm/ucentral/WifiScanEntry.java

```diff
@@ -26,10 +22,7 @@ public class WifiScanEntry extends WifiScanEntryResult {
 	 * time reference.
 	 */
 	public long unixTimeMs;
-	public Country country;
-	public QbssLoad qbssLoad;
-	public LocalPowerConstraint localPowerConstraint;
-	public TxPwrInfo txPwrInfo;
+	public InformationElements ies;
```

Comment:
- don't use the same name `ies`, it will be confusing -- `ie` or `ieContainer` or such?
- add javadoc